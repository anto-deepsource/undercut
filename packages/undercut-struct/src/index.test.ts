import { describe, expect, test } from "@jest/globals";

import * as mainExports from "./index";

describe(`Collections`, () => {
	test(`exports of the main entry should be stable`, () => {
		expect(Object.keys(mainExports).sort()).toMatchSnapshot();
	});
});
