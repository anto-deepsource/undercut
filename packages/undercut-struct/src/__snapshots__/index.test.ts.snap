// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collections exports of the main entry should be stable 1`] = `
Array [
  "Heap",
  "MaxHeap",
  "MinHeap",
  "getLeftChildIndex",
  "getParentIndex",
  "getRightChildIndex",
]
`;
